OpenUtau.Core/Classic/VoicebankLoader.cs

```diff
@@ -92,9 +94,22 @@ public class VoicebankLoader {
                 }
             }
             var enuconfigFile = Path.Combine(dir, kEnuconfigYaml);
+            var simpleEnunuFolder = Path.Combine(dir, kEnunuModelPath);
             if (File.Exists(enuconfigFile)) {
                 voicebank.SingerType = USingerType.Enunu;
-            } else {
+            } 
+            else if (Directory.Exists(simpleEnunuFolder)) {
+                if (File.Exists(Path.Combine(simpleEnunuFolder, kConfigYaml))) {
```

Comment:
1. The directory existence check is unnecessary when you already check file existence. Same for the below one.
2. When you have `if (...) { if (...) { voicebank.SingerType = ... } }`, there is a chance the first condition is true but the second is false. As a result `SingerType` will never be set. This should be avoided.